### ISSUE TITLE
"Cannot use camera white balance" error in console when using Sony files and RT compiled with gcc 4.8.x

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#2105]
Originally reported on Google Code with ID 2121

```
Some users have reported that some images display in incorrect colors, probably incorrect
white balance.
When asked to run RT from a console (Linux/OS X) or to set "verbose=true" in the "options"
file (Windows), they report seeing "Cannot use camera white balance" when opening the
raw file.
For a step-by-step of what's been tried so far, see https://code.google.com/p/rawtherapee/issues/detail?id=1636#c13
I can't reproduce, two users so far can.
I use exiv2 0.23 and so does one of the users who can reproduce (vamp898).

Vamp reported that this was caused by compiling RT using -march=bdver1 and -march=native
on his AMD FX 8150, see https://code.google.com/p/rawtherapee/issues/detail?id=1980

To summarize:
- cache has been emptied
- Neutral applied
- Input profile set to (Camera)
- RT build using buildRT and "native" proctarget

Reproduced by those affected with Sony SLT-A99V and A77.
Get raw files from:
1- http://www.ignaz.org/DSC05481.ARW (mirror: http://filebin.net/6bywnx676p )
2- http://www.rawtherapee.com/shared/play/playraw38/playraw38_tim.ARW
```

Reported by `entertheyoni` on 2013-12-12 22:38:14
